pkg/util/collections/includes_excludes.go

```diff
@@ -101,6 +101,29 @@ func (ie *IncludesExcludes) ShouldInclude(s string) bool {
 	return ie.includes.Len() == 0 || ie.includes.Has("*") || ie.includes.match(s)
 }
 
+// NamespacedShouldInclude returns whether the specified namespaced item
+// should be included or not. Excludes list has higher priority.
+// Except resources listed in excludes, other things should be included.
+func (ie *IncludesExcludes) NamespacedShouldInclude(s string) bool {
```

Comment:
This works in the current code but I don't think this is the right thing to do from OO perspective.
You end up having the instance of `IncludeExcludes` struct having inconsistent behavior.

I think it would be possible to create an interface like 
```go
interface includeChecker {
    ShouldInclude(s string) bool
}
```
And composite different implementations with common methods to one struct, then assign it to backupRequest.

Reply:
Modified.